content/workshops/openshift33/exercise1.10.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Exercise 1.10 - Deploying Java Code on JBoss EAP
+workshops: openshift33
+workshop_weight: 200
+layout: lab

```

Comment:
Hmm.  Not sure why this was the case as I have exclusively used Atom on Fedora.  There were four or five files encoded with CRLF which I changed to LF.  I believe this is resolved, however a few quick web searches did not yield a way to visualize the line ending encoding.